.babelrc

```diff
@@ -1,5 +1,6 @@
 {
-  "presets": ["react", "es2015", "stage-0"],
+  "presets": ["react", "env", "stage-0"],

```

Comment:
Since you are using preset-env, you really don't need stage-0 again as preset-env takes care of everything